### PR TITLE
websites.yml: Add support for asciinema

### PR DESCRIPTION
**Pull request by margobra8**
This adds support for asciinema username
check and implements username patterns.

Closes: https://github.com/manu-chroma/username-availability-checker/issues/51

### CODE REVIEW

**Comment by manu-chroma** [comment]
avatar not available for asciicinema profiles? @margobra8

**Comment by seeeturtle** [comment]
and you should put in alphabetic order.

File: websites.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ urls:
   opensuse: https://connect.{w}.org/pg/profile/{u}
   pinterest: https://in.{w}.com/{u}
   tumblr: https://{u}.{w}.com
+  asciinema: https://{w}.org/~{u}
```
Comment (seeeturtle) [inline websites.yml]: you should put in alphabetic order. and modify below ones too.

**Comment by margobra8** [comment]
@manu-chroma yes but I can’t figure out how to scrape it. In the html it shows as an `img` object with `avatar` class.

**Comment by seeeturtle** [comment]
umm... @manu-chroma it's also gravatar. it would be support if close this https://github.com/manu-chroma/username-availability-checker/issues/37 issue

**Comment by margobra8** [comment]
Is it ok we merge just username support and then later support the avatar with gravatar?

File: websites.yml (hunk 1 above)
Comment (manu-chroma) [reply websites.yml]: @margobra8 pls fix this

**Comment by manu-chroma** [comment]
@margobra8 sure. pls address: https://github.com/manu-chroma/username-availability-checker/pull/65#discussion_r160022308

**Pull request merged**